SourceKitStressTester/Sources/StressTester/StressTester.swift

```diff
@@ -152,6 +152,9 @@ public class StressTester {
     var actions = generator
       .generate(for: tree)
       .filter { action in
+        if let offsetFilter = options.offsetFilter, offsetFilter != action.offset {
```

Comment:
Was looking back at this since I needed to reproduce locally. Shouldn't we still be allowing `replaceText` so that we can match modifications (ideally we'd merge the consecutive replaces but... 🤷)?

Reply:
Oh, we absolutely do. I just didn’t think about it because I only needed to reproduce failures in unmodified files.